CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+# 0.8.0 (17 August 2021)
+
+This release adds support for more SwiftUI types and modifiers, and fixes bugs. Including, but not
+limited to:
+
+- `Toolbar` type and `toolbar` modifier
+- `ProgressView` type
+- `Animation` and related types and modifiers
+- `opacity`, `scaleEffect`, `aspectRatio`, `controlSize`, and `controlProminence` modifiers
```

Comment:
```suggestion
- `opacity`, `scaleEffect`, `aspectRatio`, and `controlSize` modifiers
```
`controlProminence` was removed in #435 and replace with `BorderedProminentButtonStyle`